File: Object Oriented Programming/Java/Pizza.java
public class Pizza {
    
    // Encapsulation - restricting access to objects internal state. 
    private String size;
    private double cheese;
    private double pepperoni;
    private double veggies;

    // Default Constructor
    public Pizza(){
        this.size = "small";
        this.cheese = 0;
        this.pepperoni = 0;
        this.veggies = 0;
    }

    // Overloaded Constructor
    public Pizza(String size, double cheese, double pepperoni, double veggies) {
        this.size = size;
        this.cheese = cheese;
        this.pepperoni = pepperoni;
        this.veggies = veggies;
    }

    // Encapsulation - Exposing only what is necessary through public methods.
    public String getSize(){
        return this.size;
    }

}

File: Object Oriented Programming/Python/Pizza.py
class Pizza:
    def __init__(self, size, cheese, pepperoni, veggies):
        self.size = size
        self.cheese = cheese
        self.pepperoni = pepperoni
        self.veggies = veggies

    def calcCost(self):
        total = 0
        total += 10 if self.size == "small" else 15 if self.size == "medium" else 20 if self.size == "large" else 0
        return total + self.cheese + self.pepperoni + self.veggies

    def __str__(self):
        ret = "Size: " + self.size + "\n"
        ret += "Cheese: " + str(self.cheese) + "\n"
        ret += "Pepperoni: " + str(self.pepperoni) + "\n"
        ret += "Veggies: " + str(self.veggies) + "\n"
        ret += "Cost: " + str(self.calcCost()) + "\n"

        return ret



class PizzaOrder:
    def __init__(self, num_pizzas):
        self.num_pizzas = num_pizzas
        

    def __str__(self):
        pass